packages/odyssey-storybook/src/components/Field/Field.docgen.tsx

```diff
@@ -0,0 +1,18 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import React, { forwardRef } from "react";
+import type { FieldProps } from "@okta/odyssey-react";
+
+const Source = forwardRef<HTMLDivElement, FieldProps>(() => <></>);
```

Comment:
lets rename this to something more meaningful like `FieldDocGen` and export it below on line 18 - I wanted to make the same change for `Box.

Also, no need for the `forwardRef` here, all we need is a function component that accepts the same props as `Field` and returns `null`, we don't even need the react fragment

Reply:
Why do these docgen files exist at all? They feel completely unnecessary on the surface, does this have to do with complexity in our TypeScript implementation?

Reply:
@arnoldsandoval the [tool](https://github.com/styleguidist/react-docgen-typescript) we're using to create prop tables in our docs doesn't understand some of the component composition patterns we're using, forwardRefWithStatics being one, polymorphic props being another. Rather than trying to continue to tinker with our component source we're going to setup docgen only components within `@okta/odyssey-storybook` to make parsing out prop tables straightforward for the parser. Open to other suggestions or ideas too